### PR TITLE
Add kubeval to our kubectl image

### PR DESCRIPTION
**Pull request by axelpavageau**

### CODE REVIEW

File: kubectl/Dockerfile [hunk 1]
```diff
@@ -27,3 +28,7 @@ RUN wget https://raw.githubusercontent.com/ahmetb/kubectx/v${KUBECTX_VERSION}/ku
 RUN wget https://github.com/zegl/kube-score/releases/download/v${KUBESCORE_VERSION}/kube-score_${KUBESCORE_VERSION}_linux_amd64 && \
     chmod +x kube-score_${KUBESCORE_VERSION}_linux_amd64 && \
     mv kube-score_${KUBESCORE_VERSION}_linux_amd64 /usr/bin/kube-score
+
+RUN wget https://github.com/instrumenta/kubeval/releases/download/${KUBEVAL_VERSION}/kubeval-linux-amd64.tar.gz && \
+    tar xf kubeval-linux-amd64.tar.gz && \
+    mv kubeval /usr/local/bin
```
Comment (mremi) [inline kubectl/Dockerfile]: No need chmod?

Comment (axelpavageau) [reply kubectl/Dockerfile]: nope ```>> Running test: kubeval --version
247Version: 0.14.0
248Commit: eae975a0031c57f01976474f428861f50cfc551e
249Date: 2019-09-01T10:07:06Z```

File: config.yml [hunk 2]
```diff
@@ -154,11 +154,13 @@ kubectl:
         - kustomize version
         - kubectx
         - kube-score version
+        - kubeval --version
     build_args:
       KUBECTL_VERSION: 1.15.1
       KUSTOMIZATION_VERSION: 2.0.3
       KUBECTX_VERSION: 0.6.3
       KUBESCORE_VERSION: 0.7.1
+      KUBEVAL_VERSION: 0.14.0
```
Comment (mremi) [inline config.yml]: Could you please sort by name? And so in Dockerfile

Comment (axelpavageau) [reply config.yml]: done

**Pull request merged**